staging/src/k8s.io/apiextensions-apiserver/pkg/registry/customresource/strategy.go

```diff
@@ -159,6 +160,11 @@ func copyNonMetadata(original map[string]interface{}) map[string]interface{} {
 	return ret
 }
 
+const (
+	//TODO: pick the RuntimeCELCostBudget
+	RuntimeCELCostBudget = math.MaxInt64
```

Comment:
Would it be possible to consolidate all our constants to a single file somewhere? That might help self-document how all the cost limits work and related to each other.

cc @DangerOnTheRanger

Reply:
Make sense. Maybe we have all related const in compilation.go? @DangerOnTheRanger What would you think?

Reply:
I have moved the related cost to `compilation.go`. Let me know if other place is preferred

Reply:
I think `compilation.go` is good; part of me wonders if maybe a separate file in the same package might be a good idea at some point, but I don't think we have enough constants for that just yet.